src/app/workflow/version-modal/version-modal.service.spec.ts

```diff
@@ -87,9 +87,10 @@ describe('Service: version-modal.service.ts', () => {
     expect(regexp.test('/Dockstore.cw')).toBeFalsy();
   });
 
-  it('regex should still be valid in v mode', () => {
-    for (const [, pattern] of Object.entries(validationDescriptorPatterns)) {
-      expect(new RegExp(pattern, 'v')).toBeTruthy();
-    }
-  });
+  // TODO: https://ucsc-cgl.atlassian.net/browse/SEAB-6612 someone more well versed than me should investigate whether this is known breakage with newer node versions
```

Comment:
Ticket doesn't look quite right, maybe you meant this one? https://ucsc-cgl.atlassian.net/browse/SEAB-7158